index.js

```diff
@@ -470,7 +474,11 @@ module.exports = function (log, indexesPath) {
       },
       end: () => {
         const count = seq // incremented at end
-        debug(`time: ${Date.now() - start}ms, total items: ${count}`)
+        debug(
+          `log.stream #${logstreamId} done ${seq - startSeq} records in ${
+            Date.now() - start
+          }ms`
+        )
```

Comment:
Note that previously this was reporting `count` which was wrong because that's not the number of records the log.stream went through, it's just the "last seq" when the stream ended. `createIndexes()` doesn't have this issue, only `updateIndex()`